common/f3d_materials.md

```diff
@@ -0,0 +1,193 @@
+# F3D Material Overview
+
+F3D materials offer an accurate way of recreating N64 graphics within blender. The materials contain all the inputs and outputs that N64 graphics using Nintendo's standard "Fast 3D" material system uses, with almost no concessions with regards to visual accuracy.
+
+F3D materials are can be modified in either a simple or full UI. Using the simplified UI will allow you to select from preset options, and change the applicable input variables as needed, while the full UI will unlock all F3D options available. Each section within the full UI represent an important chunk of the F3D material system, and should be understood to make full use of the N64s graphical capabilities.
+
+:::::{tab-set}
+
+::::{tab-item} Color Combiner & Sources
```

Comment:
I do not like this tab thing, it is anti-ctrl F and there's way too much below it
I'd suggest splitting into pages instead